### PR TITLE
Implement frequency filter

### PR DESCRIPTION
**Pull request by ananastii**
The new filter allows to choose which results to show based on their frequency, and also limits the number of visible results

### CODE REVIEW

File: data/js/filter.js [hunk 1]
```diff
@@ -0,0 +1,74 @@
+/* пересобирает результат так, что каждой частоте соответствует массив результатов
```
Comment (akutuzov) [inline data/js/filter.js]: All comments must be in English

File: data/js/filter.js [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+    });
+  };
+  
+  // проверяем чекбоксы при запуске страницы и при любом клике
+  $(document).ready(function(){
+    const RESULTS = JSON.parse($("#result").data("result").replace(/'/g, '"'));
+    const FREQUENCIES = ['high', 'mid', 'low'];
+    const MODELS = Object.keys(RESULTS);
+    const FILTER = 'freq';
+    const MAXNUM = ($('title').text() == 'WebVectors: Semantic Calculator' || $('title').text() == 'WebVectors: Семантический калькулятор') ? 5 : 10
```
Comment (akutuzov) [inline data/js/filter.js]: Why these hardcoded strings here?

**Review by akutuzov — CHANGES_REQUESTED** [review]

File: data/js/filter.js (hunk 2 above)
Comment (ananastii) [reply data/js/filter.js]: There are some values which I can't extract from the server reply - maximal number of visible results, the name of the filter, or can - frequencies - but with some strange manipulations. Others  I extract from html once and then use in functions without changes. These are constants for each function, that's why they are collected together and their names are upper-cased

Comment (ananastii) [reply data/js/filter.js]: I can receive [high, mid, low] from checkbox, but for me it looks like just another level of hardcoding: 
 I "hardcode" this values into checkbox, not receive from the server. But I'm newby, so maybe to receive all filters and their values from the backend is just more beautiful, not more useful :)

**Comment by ananastii** [comment]
Single quote in .htmls (data-result ='{{ frequencies|tojson }}') is not a mistake. It is required to transfer and parse data with double qoute (frequencies) correctly (http://jinja.pocoo.org/docs/2.10/templates/#tojson).

The code is tested on data contains words with ' ("д'альбро")

**Review by akutuzov — APPROVED** [review]

**Review by lizaku — APPROVED** [review]
looks good

**Pull request merged**